google_looker_instance/looker_instance_enterprise_full/main.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START looker_google_looker_instance_enterprise_full]
+# Creates an Enterprise edition Looker (Google Cloud core) instance with full, Private IP functionality.
+resource "google_looker_instance" "main" {
```

Comment:
I get an error on the `google_looker_instance` resource:

```
│ Error: Error creating Instance: googleapi: Error 400: Not a valid KMS resource name: 'looker-kms-key'
│ 
│   with google_looker_instance.main,
│   on main.tf line 19, in resource "google_looker_instance" "main":
│   19: resource "google_looker_instance" "main" {
│ 
```

My guess is that we might have to create this resource and reference it here

Reply:
Done.